app/views/citizens/own_homes/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1 class="govuk-heading-xl"><%= content_for(:page_title) %></h1>
+
+<%= render partial: 'shared/forms/error_summary', locals: { model: @form } %>
+
+    <%= form_with(model: @form, url: form_path, method: :patch, local: true) do |form| %>
+
+      <div class="govuk-!-padding-bottom-8"></div>
+
+      <% error = form.object.errors[:own_home].first %>
+      <% group_error_class = error ? 'govuk-form-group--error' : '' %>
+      <% input_error_class = error ? 'govuk-input--error' : '' %>
+
+      <div class="govuk-form-group <%= group_error_class %>">
+        <div class="govuk-radios">
+          <%= content_tag(:span, error, class: 'govuk-error-message', id: 'own_home') if error %>
+          <div class="govuk-radios__item">
+            <%= form.radio_button 'own_home', 'mortgage', class: "govuk-radios__input #{input_error_class}" %>
+            <%= form.label 'own_home_mortgage', action_t('.own_home.mortgage'), class: 'govuk-label govuk-radios__label' %>
+          </div>
+          <div class='govuk-radios__item'>
+            <%= form.radio_button 'own_home', 'owned_outright', class: "govuk-radios__input #{input_error_class}" %>
+            <%= form.label 'own_home_owned_outright', action_t('.own_home.owned_outright'), class: 'govuk-label govuk-radios__label' %>
+          </div>
+          <div class='govuk-radios__item'>
+            <%= form.radio_button 'own_home', 'no', class: "govuk-radios__input #{input_error_class}" %>
+            <%= form.label 'own_home_no', action_t('.own_home.no'), class: 'govuk-label govuk-radios__label' %>
+          </div>
+        </div>
+      </div>
+
+      <div class="govuk-!-padding-bottom-6"></div>
```

Comment:
If you rebase from master you should be able to use the new `govuk_collection_radio_buttons` compoment. you could simplify this file a lot:
```
<% content_for :navigation do %>
  <%= link_to t('generic.back'), citizens_consent_path(id: session[:current_application_ref]), class: 'govuk-back-link', id: 'back' %>
<% end %>

<% content_for(:page_title) { t('.h1-heading') } %>

<%= render partial: 'shared/forms/error_summary', locals: { model: @form } %>

<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
    <%= form_with(model: @form, url: form_path, method: :patch, local: true) do |form| %>

      <%
        options = [
          { value: :mortgage,       label: action_t('.own_home.mortgage') },
          { value: :owned_outright, label: action_t('.own_home.owned_outright') },
          { value: :no,             label: action_t('.own_home.no') }
        ]
      %>

      <%= form.govuk_collection_radio_buttons(:own_home, options, :value, :label, title: content_for(:page_title)) %>

      <div class="govuk-!-padding-bottom-6"></div>

      <%= form.submit t('generic.continue'), id: 'continue', class: 'govuk-button' %>
    <% end %>
  </div>
</div>

```

Reply:
Thank you!